app/_locales/es/messages.json

```diff
@@ -24,7 +24,7 @@
     "message": "Datos de aprobación"
   },
   "approvalDataDescription": {
-    "message": "Borre los datos del sitio web aprobado para que todos los sitios deban volver a solicitar la aprobación."
+    "message": "Borrar los datos para que todos los sitios deban volver a solicitar la aprobación."
```

Comment:
What you think about: "Borrar la información privada de modo que todos los sitios deban volver a requeir acceso para acceder a los datos de la cuenta."?

Reply:
well, 
that sounds good -- I assume you mean 'requerir'. It's not what the English says, but in the end it means the same thing:

"Clear privacy data so all websites must request access to view account information again."